k8s/upgrade/src/bin/upgrade-job/helm/upgrade.rs

```diff
@@ -82,6 +93,7 @@ impl HelmUpgradeBuilder {
         );
         let release_name = self.release_name.clone().unwrap();
         let namespace = self.namespace.clone().unwrap();
+        let values = self.values.clone().unwrap();
```

Comment:
We should handle error instead of unwrap.
We could either:
1. use like builder-like pattern and return error here
2. not allow them to be optional by including their setup in the new fn
3. use type-state pattern and only expose build method if they're all Some

Reply:
There is a line just above which handles the error in case of None.
```
 ensure!(
            self.release_name.is_some() && self.namespace.is_some(),
            HelmUpgradeOptionsAbsent
        );
```
And for set values i changed it to unwrap_or_default 
`let values = self.values.clone().unwrap_or_default();`

Reply:
hmm a more idiomatic way of doing it would not be this way but rather something like this:
let release_name = self.release_name.ok_or(Err::HelmUpgradeOptionsAbsent{})?;
where now you have a variable which has be validated and ready to use.